src/Routes/UpcomingPage/UpcomingPage.tsx

```
@@ -14,14 +14,27 @@ const UpcomingPage = () => {
-    <div>
-      View the latest updates on upcoming Red Hat Enterprise Linux features, tailored to your systems. Add systems
-      to{' '}
-      <a href="https://console.redhat.com/insights/inventory" target="_blank" rel="noopener noreferrer">
-        Inventory
-      </a>{' '}
-      to view only upcoming features relevant to your organization.
-    </div>
+    <Content>
+      <p>
+        View the latest updates on upcoming Red Hat Enterprise Linux features, tailored to your systems. Add systems
+        to{' '}
```

ESLint

[error] Replace `·systems⏎·······` with `⏎········systems`

(prettier/prettier)